tool/sel4coreplat/__main__.py

```diff
@@ -1594,7 +1594,7 @@ def main() -> int:
     kernel_config = KernelConfig(
         word_size = 64,
         minimum_page_size = kb(4),
-        paddr_user_device_top = (1 << 40),
+        paddr_user_device_top = (1 << 44),
```

Comment:
This might break the untypeds for any CPU that support less than 44 bits of physical address, since the kernel will create device untypeds up to `CONFIG_PADDR_USER_DEVICE_TOP`. Is there a way to temporarily supply this info from the BoardConfig instead? (The real solution is the FIXME ;-) )